src/els_document_highlight_provider.erl

```diff
@@ -45,20 +45,17 @@ find_highlights(Uri, #{ kind := Kind
                       }) when Kind =:= application;
                               Kind =:= implicit_fun;
                               Kind =:= function;
-                              Kind =:= variable;
                               Kind =:= export_entry ->
   Key = case Id of
           {F, A}    -> {els_uri:module(Uri), F, A};
-          {M, F, A} -> {M, F, A};
-          V         -> {els_uri:module(Uri), V, 0}
+          {M, F, A} -> {M, F, A}
```

Comment:
what about matching `string () | atom ()`?  Or is that excluded based on the guard clauses?

Reply:
Exactly, those cases are excluded because of the filtering done in the guard.